docs/docs/cmd/aad/administrativeunit/administrativeunit-get.mdx

```diff
@@ -0,0 +1,98 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# aad administrativeunit get
+
+Gets information about a specific administrative unit
+
+## Usage
+
+```sh
+m365 aad administrativeunit get [options]
+```
+
+## Options
+
+```md definition-list
+`-i, --id [id]`
+: The id of the administrative unit. Specify either `id` or `displayName` but not both.
+
+`-n, --displayName [displayName]`
+: The display name of the administrative unit. Specify either `id` or `displayName` but not both.
+```
+
+<Global />
+
+## Examples
+
+Get information about the administrative unit by its id
+
+```sh
+m365 aad administrativeunit get --id 03c4c9dc-6f0c-4c4f-a4e6-0c9ed80f54c7
+```
+
+Get information about the administrative unit by its display name
+
+```sh
+m365 aad administrativeunit get --displayName 'Marketing Division'
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  {
+  "id": "0a22c83d-c4ac-43e2-bb5e-87af3015d49f",
+  "deletedDateTime": null,
```

Comment:
Let's add some extra indentation in this json object

Reply:
Added